rskj-core/src/main/java/org/ethereum/net/server/Channel.java

```diff
@@ -263,6 +280,12 @@ public void imported(boolean best) {
         stats.imported(best);
     }
 
+    @Override
+    public boolean isSnapCapable() {
+        return capabilities.stream()
```

Comment:
Capabilities shouldn't vary over time, so I won't do this filtering every time we need to check this condition. I would filter only once and store the value of the result instead of the capabilities list.

Reply:
Thanks for the suggestion! I've implemented it by adding a `isSnapCapable` variable to store the result of the check. This avoids filtering the capabilities list every time and should improve efficiency